src/components/GlobalHeader/GlobalHeader.module.css

```
@@ -11,10 +11,32 @@
 }
 
 .logo {
-    display: flex;
-    justify-content: center;
     align-items: center;
-    width: 15vh;
+    width: 18vh;
```

```suggestion
.logo {
    display: flex;
    align-items: center;
    height: 15vh;
    margin: 5vh;
}

.logo a {
    text-align: center;
}
```
These rules should align and center the logo and text